AnnoDesigner.Core/Presets/Models/BuildingInfo.cs

```diff
@@ -82,6 +82,24 @@ public class BuildingInfo : IBuildingInfo
         [DataMember(Order = 10)]
         public bool Borderless { get; set; }
 
+        /// <summary>
+        /// Length of blocked area
+        /// </summary>
+        [DataMember(Order = 11)]
+        public double BlockedAreaLength { get; set; }
+
+        /// <summary>
+        /// Length of blocked area
+        /// </summary>
+        [DataMember(Order = 12)]
+        public double BlockedAreaWidth { get; set; }
+
+        /// <summary>
+        /// Direction of blocked area
+        /// </summary>
+        [DataMember(Order = 13)]
+        public GridDirection Direction { get; set; } = GridDirection.Down;
```

Comment:
Is this the correct default value? Maybe better to use `Undefined` (see above).

Reply:
This is what is need to get the default so yeah that is correct (we can only choice the 4 given there)